### ISSUE TITLE
Spelling error in help text

### ISSUE THREAD

**Issue by fiskeren** [jotta/jotta-cli-issues#119]
Version:
```
~> jotta-cli version
-------------------------------------------
jottad executable : 
jottad appdata    : /var/lib/jottad
jottad version    : 0.6.32724
jotta-cli version : 0.6.32724
-------------------------------------------
OK
```

The help text for config set has a spelling error in the word scaninterval:
```
  jotta-cli config set scaninternval 30m     scans are now run every 30min
```

Should be:
 ```
jotta-cli config set scaninterval 30m     scans are now run every 30min
```

**Comment by Kimbsen** [comment]
Thanks for that. Will fix for the next release!

**Comment by Kimbsen** [comment]
We've just released version 0.7.35160 which should fix this issue.
